### ISSUE TITLE
Mickey Mouse Voice

### ISSUE THREAD

**Issue by rogerknop** [jitsi/jitsi-meet#6326]
Hello Jitsi Team!

## Description
Sometimes the voice sounds like mickey mouse. There was a similar issue in 2016: https://community.jitsi.org/t/jitsi-dev-jitsi-jitsi-first-call-to-windows-from-linux-ends-up-in-mickey-mouse-speech-267/10990
Problem is, that it can't be reproduced. Sometimes this problem occurs directly when starting the meeting and sometimes it is starting with normal voice and then the issue is slowly increasing.
Sounds really funny but at the end the meeting need to be skipped.

# Environment details
As far as I can say it only occurs if one side is using jitsi-meet app with electron for remote control.
And the other side is on jitsi.org in the web.

Regards, Roger

**Comment by Walnussbaer** [comment]
I can confirm that this can happen, we just experienced it yesterday

**Comment by dtrossiter** [comment]
We experienced this today on meet.jit.si. The participant was able to get around it, but I don't know what he changed.

**Comment by JimHurley** [comment]
I am the user that drossiter mentioned in the previous post. I've had this issue on an off for some time now. I've had about 4 meetings in a row without the problem and I've been able to at times to resolve the problem by restarting my OS. Today I was using Linux Mint Cinnamon edition and nothing worked. I rebooted my machine into Windows 11 and jitsi worked using edge. I had tried edge when I was in Linux. We have another member of the club that is a Windows user and has had the problem in the past, but not recently. I have had the problem using Windows in the past and I believe it resolved when I switched to Linux. I always use the same dual boot machine 11 generation intel I5, with 16 GB of ram and nvme drive. The club member that has had the problem in the past lives in Sequim Washington and I live in Newcastle Wyoming.
